source/apiVolontaria/volunteer/tests/tests_model_Participation.py

```diff
@@ -120,3 +126,16 @@ def test_create_participation_missing_standby(self):
             user=self.user,
             event=self.event,
         )
+
+    def test_delete_participation_of_event_already_begin(self):
+        """
+        Ensure we can't delete a participation if the event is already begin
+        """
+
+        with self.assertRaises(Exception) as context:
+            self.participation.delete()
+
+        self.assertTrue(
+            'You can\'t delete an application if the associated '
```

Comment:
Application? Shouldn't it be `participation`?

Reply:
Oh yes, will be better!